sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeOperationExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.AI.FormRecognizer.Models
```

Comment:
Should this class be in the models namespace?

Reply:
Will move to `Azure.AI.FormRecognizer` to be visible without including the Models namespace.